userProfile/index.mjs

```diff
@@ -0,0 +1,84 @@
+import express from 'express'
+import * as utils from '../utilities/shared.mjs'
+import * as service from './userProfile.mjs'
+import cors from 'cors'
+
+let router = express.Router()
+router.use(
+  cors({
+    "methods" : "GET",
+    "allowedHeaders" : [
+      'Content-Type',
+      'Content-Length',
+      'Allow',
+      'Authorization',
+      'Location',
+      'ETag',
+      'Connection',
+      'Keep-Alive',
+      'Date',
+      'Cache-Control',
+      'Last-Modified',
+      'Link',
+      'X-HTTP-Method-Override'
+    ],
+    "exposedHeaders" : "*",
+    "origin" : "*",
+    "maxAge" : "600"
+  })
+)
+
+
```

Comment:
Why the extra newline whitespace?  If it doesn't serve a purpose just reduce it to a single newline.  Do this throughout this file.

Reply:
updated resolved.